sbin/build.sh

```diff
@@ -846,11 +861,13 @@ configureCommandParameters
 buildTemplatedFile
 executeTemplatedFile
 
-printJavaVersionString
+if [[ "${BUILD_CONFIG[MAKE_EXPLODED]}" != "true" ]]; then
```

Comment:
Isn't this true by default as we would have exited previously in the other if case?

Reply:
no, in a normal build (without `--make-exploded` or `--assemble-exploded-image`) the steps would need to be run